themes/default/content/templates/container-service/aws/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Deploy an ECS Container Service on AWS
+meta_desc: The Containers template makes it easy to deploy an ECS containerized application on AWS with Pulumi and Amazon ECS.
```

Comment:
```suggestion
meta_desc: The Container Service template makes it easy to deploy a containerized application on AWS with Pulumi and Amazon ECS.
```

Reply:
But the thing here really isn't the containizerized app right? The thing here is it deploys a container service with an load balancer (and it just so happens that there is an application running atop)